src/main/java/com/lob/protocol/response/CheckResponse.java

```diff
@@ -95,8 +101,16 @@ public String getUrl() {
         return url;
     }
 
-    public TrackingResponse getTracking() {
-        return tracking;
+    public String getCarrier() {
+        return carrier;
+    }
+
+    public String getTrackingNumber() {
+        return trackingNumber;
+    }
+
+    public List<TrackingEventResponse> getTrackingEvents() {
+        return trackingEvents;
```

Comment:
Since this `List` is a Java object, returning it directly allows the end user to modify it and that would effect this object directly. The way to handle this is to make a copy of the object with `defensiveCopy` before return it (like what `getThumbnails` does).